birdhouse/components/cowbird/docker-compose-extra.yml

```diff
@@ -30,7 +30,7 @@ services:
     volumes:
       - ./components/cowbird/config/cowbird/config.yml:/opt/local/src/cowbird/config/cowbird.yml
       - ./components/cowbird/config/cowbird/cowbird.ini:/opt/local/src/cowbird/config/cowbird.ini
-      - "${DATA_PERSIST_ROOT}/user_workspaces:/user_workspaces"
+      - "${DATA_PERSIST_ROOT}/${USER_WORKSPACES}:/${USER_WORKSPACES}"
```

Comment:
Maybe Cowbird will actually need `${DATA_PERSIST_ROOT}/` mount as well in order to generate the links. To be tested...